ipa-multipoint/ipa_multipoint_jni/Cargo.toml

```diff
@@ -22,7 +22,7 @@ repository = "https://github.com/hyperledger/besu-native"
 edition = "2018"
 
 [dependencies]
-ffi_interface = { git = "https://github.com/crate-crypto/rust-verkle", rev = "7688f0aedfb147d3d391abfe8495e46c46d72ce0" }
+ffi_interface = { git = "https://github.com/crate-crypto/rust-verkle", branch = "master" }
```

Comment:
I prefer to have a specific commit in order to build the same thing everytime

Reply:
No problem, done.